app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -423,6 +423,14 @@ fun parseUrl(url: String): String? {
 fun openLink(url: String, ctx: Context, useCustomTab: Boolean, usePrivateTab: Boolean) {
     val url = parseUrl(url) ?: return
 
+    if (url.startsWith("@")) {
```

Comment:
Following #583, I don't think this section triggers. `parseUrl` will return a standard https link. This should probably be rewritten to check if a link appears to be a user or community link, and then handle it.

Reply:
I got excited and ended up doing that: https://github.com/twizmwazin/jerboa/tree/feat/implement-lemmy-links

Reply:
I looked through your changes, and you may want to consider setting `navController` outside of the `MarkdownHelper.init()` so it can easily be used in previews. What I did was make it nullable and made a setter for it, and then check it before passing it on to openLinks.

Just an idea, I'm certainly no expert here. 😄

Reply:
Yeah I think you're right, would welcome a PR for it.